### ISSUE TITLE
(eks): Newly introduced default timeout of helm chart breaks 'wait' option

### ISSUE THREAD

**Issue by zxkane** [aws/aws-cdk#8713]
After upgrading cdk to latest `1.46.0`, deploying a helm chart to EKS failing on timeout.

### Reproduction Steps

Deploy a helm chart without specifying `wait` and `timeout` in slow network env(such as China regions) like below,

```typescript
cluster.addChart('Nexus3', {
      chart: 'sonatype-nexus',
      repository: 'https://oteemo.github.io/charts/',
      namespace: nexus3Namespace,
      release: nexus3ChartName,
      version: nexus3ChartVersion,
      values: { ... }
}
```
See complete app [here](https://github.com/zxkane/cdk-collections/blob/master/eks/sonatype-nexus3/lib/sonatype-nexus3-stack.ts#L334).

The deployment fails on timeout waiting the chart is completely deployed.
```
9:09:30 PM | CREATE_FAILED        | Custom::AWSCDK-EKS-HelmChart          | MyK8SCluster/chart...3/Resource/Default
Failed to create resource. Error: b'Release "nexus3" does not exist. Installing it now.\nError: timed out waiting for th
e condition\n'
at invokeUserFunction (/var/task/framework.js:85:19)
at process._tickCallback (internal/process/next_tick.js:68:7)
```

It should be introduced by #8338 that does not honor the default `wait` value `false`. Looks like explicit `--timeout` enables `--wait` switch as well.

> --timeout duration             time to wait for any individual Kubernetes operation (like Jobs for hooks) (default 5m0s)
> --wait                         if set, will wait until all Pods, PVCs, Services, and minimum number of Pods of a Deployment, StatefulSet, or ReplicaSet are in a ready state before marking the release as successful. It will wait for as long as --timeout

### Environment

  - **CLI Version      :** 1.46.0
  - **Framework Version:**
  - **Node.js Version:** 
  - **OS               :**
  - **Language (Version):** 

### Other

--- 

This is :bug: Bug Report

**Comment by eladb** [comment]
Thanks for reporting

**Comment by eladb** [comment]
Resolved by #8548
